webapp/package.json

```diff
@@ -13,6 +13,7 @@
     "clean": "npm run clean --workspaces --if-present"
   },
   "devDependencies": {
+    "@types/crypto-js": "4.1.1",
```

Comment:
@gomessguii This should actually be added to `webapp/channels/package.json`. This one at the top level is just for the scripts to run all the different packages in the monorepo

Reply:
I just ran npm install, I wasn't aware of the directories scheme. I'll fix it.

Reply:
Ah, I see. I should've recommended running `npm add --workspace=channels`. I've submitted a PR to explain that properly since I realized I never explained how to do that in any of our docs